tensorflow/contrib/seq2seq/python/ops/beam_search_decoder.py

```diff
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 import collections
+import numpy as np
```

Comment:
please remove this (already imported below). I think this may caused by the rebase.